libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/factory/RouteLegProgressFactory.kt

```diff
@@ -28,6 +28,7 @@ object RouteLegProgressFactory {
      * is currently on
      * @param upcomingStep Next/upcoming step immediately after the current step. If the user is on the last step
      * on the last leg, this will return null since a next step doesn't exist
+     * @param geometryIndex Current index in a leg geometry
```

Comment:
The index represents the geometry point right in front of us, doesn't it? It'd be great to specify it in the docs.